packages/cli/src/editor/components/RenderModal/RenderModalData.tsx

```diff
@@ -39,6 +51,8 @@ export const RenderModalData: React.FC<{
 		return composition.schema.safeParse(inputProps);
 	}, [composition.schema, inputProps]);
 
+	const cliProps = getInputProps();
+	console.log(Object.keys(cliProps).length);
```

Comment:
I thought you would not commit it this time 😛

Reply:
I thought so too👀😂